### ISSUE TITLE
Best practice to run Mask R-CNN in parallel

### ISSUE THREAD

**Issue by maxfrei750** [pytorch/vision#1552]
What ist the best practice to run Mask R-CNN in parallel?

@fmassa wrote in #1255 

> The current code assumes that you are using 1 GPU per process, with DistributedDataParallel.

Is this information up-to-date?

**Comment by fmassa** [comment]
We use 1 GPU per python process, but that doesn't mean that we can't use more than 1 GPU for training / inference.

Best practice is to run the training code via
```
python -m torch.distributed.launch --nproc_per_node=8 --use_env train.py
```
